### PR TITLE
Create seed script

### PR DESCRIPTION
**Pull request by andrewgeorgemitchell**
Seed script creates csv files for all my tables/graph node labels

### CODE REVIEW

File: seeding/seed.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const seedSongs = require('./helpers/seedSongs');
+const seedUsers = require('./helpers/seedUsers');
+const seedPlaylists = require('./helpers/seedPlaylists');
+const seedAlbums = require('./helpers/seedAlbums');
+
+const OoM = 10;
```
Comment (anlee3) [inline seeding/seed.js:6]: This variable is not very clear.

**Review by anlee3 — APPROVED** [review]
LGTM. Good job!

File: package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     "start": "node --experimental-modules server/index.mjs",
     "server": "nodemon --experimental-modules server/index.mjs",
     "createDB": "mysql -u root  < db/schema.sql",
-    "seedDB": "node --experimental-modules db/seed.mjs",
+    "seed": "node seeding/seed.js",
     "elastic": "npm run-script build && npm run-script server"
   },
```
Comment (spraguejdev) [inline package.json:21]: @andrew121212 looks good.

File: seeding/downloadImages.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Basic while loop
+counter=1
+while [ $counter -le 1000 ]
+do
+touch "/Users/Andrew/Desktop/SDC/${counter}example.jpg"
+curl http://lorempixel.com/400/200/ >  "/Users/Andrew/Desktop/SDC/${counter}example.jpg"
+((counter++))
+done
+echo All done
```
Comment (spraguejdev) [inline seeding/downloadImages.sh:10]: @andrew121212  nice!

File: seeding/helpers/seedAlbums.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      }
+      // eslint-disable-next-line no-await-in-loop
+      await writeToFile(filePath, batch, i);
+      batch = [];
+    }
+    resolve();
+  } catch (error) {
+    reject(error);
+  }
+});
```
Comment (spraguejdev) [inline seeding/helpers/seedAlbums.js:33]: @andrew121212 Looks good.

File: seeding/helpers/seedUsers.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      }
+      // eslint-disable-next-line no-await-in-loop
+      await writeToFile(filePath, batch, i);
+      batch = [];
+    }
+    resolve();
+  } catch (error) {
+    reject(error);
+  }
+});
```
Comment (spraguejdev) [inline seeding/helpers/seedUsers.js:33]: @andrew121212 all your scripts are good.

File: seeding/seed.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  console.timeEnd('seedUsers');
+
+  console.time('seedPlaylists');
+  await seedPlaylists(`${__dirname}/csv/playlists.csv`, OoM);
+  console.timeEnd('seedPlaylists');
+
+  console.time('seedAlbums');
+  await seedAlbums(`${__dirname}/csv/albums.csv`, OoM);
+  console.timeEnd('seedAlbums');
+})();
```
Comment (spraguejdev) [inline seeding/seed.js:24]: @andrew121212 I like your await style.

**Review by spraguejdev — COMMENTED** [review]
@andrew121212 Good job. Everything looks good. Can't wait to see your db when finished.

**Pull request merged**